radixdlt/src/main/java/com/radixdlt/CerberusModule.java

```diff
@@ -32,45 +34,74 @@
 import com.radixdlt.consensus.liveness.PacemakerImpl;
 import com.radixdlt.consensus.liveness.PacemakerRx;
 import com.radixdlt.consensus.liveness.ProposerElection;
-import com.radixdlt.consensus.safety.QuorumRequirements;
 import com.radixdlt.consensus.safety.SafetyRules;
-import com.radixdlt.consensus.safety.SingleNodeQuorumRequirements;
 import com.radixdlt.consensus.tempo.Scheduler;
 import com.radixdlt.consensus.tempo.SingleThreadedScheduler;
+import com.radixdlt.consensus.validators.Validator;
+import com.radixdlt.consensus.validators.ValidatorSet;
 import com.radixdlt.crypto.ECDSASignatures;
 import com.radixdlt.crypto.ECKeyPair;
 import com.radixdlt.engine.RadixEngine;
+import com.radixdlt.properties.RuntimeProperties;
 import com.radixdlt.universe.Universe;
+import java.util.Collections;
+import java.util.Objects;
 import org.radix.logging.Logger;
 import org.radix.logging.Logging;
 
 import java.util.concurrent.Executors;
+import org.radix.network2.addressbook.AddressBook;
 
 public class CerberusModule extends AbstractModule {
 	private static final Logger log = Logging.getLogger("Startup");
 
+	private final RuntimeProperties runtimeProperties;
+
+	public CerberusModule(RuntimeProperties runtimeProperties) {
+		this.runtimeProperties = Objects.requireNonNull(runtimeProperties);
+	}
+
 	@Override
 	protected void configure() {
 		// dependencies
-		bind(QuorumRequirements.class).to(SingleNodeQuorumRequirements.class);
 		bind(Scheduler.class).toProvider(SingleThreadedScheduler::new);
 		bind(PacemakerRx.class).to(PacemakerImpl.class);
 		bind(Pacemaker.class).to(PacemakerImpl.class);
 		bind(SafetyRules.class).in(Scopes.SINGLETON);
 	}
 
+	@Provides
+	@Singleton
+	private EpochRx epochRx(
+		@Named("self") ECKeyPair selfKey,
+		AddressBook addressBook
+	) {
+		final int fixedQuorumSize = Integer.parseInt(runtimeProperties.get("consensus.fixed_quorum_size", "1"));
+		return new BasicEpochRx(selfKey.getPublicKey(), addressBook, fixedQuorumSize);
+	}
+
+	@Provides
+	@Singleton
+	private ValidatorSet validatorSet(
+		@Named("self") ECKeyPair selfKey
+	) {
+		return ValidatorSet.from(Collections.singleton(Validator.from(selfKey.getPublicKey())));
+	}
+
 	@Provides
 	@Singleton
 	private ProposerElection proposerElection(
 		@Named("self") ECKeyPair selfKey
 	) {
-		return new Dictatorship(selfKey.getUID());
+		return new Dictatorship(selfKey.getPublicKey());

```

Comment:
I think Dictatorship implementation can disappear altogether now.

Reply:
Might still be useful for single node setups. But I guess those can just be represented as a multi-node setup with just one validator